Lesson5/Task1/Rectangle.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Lesson5
+{
+    class Rectangle
+    {
+        private int side1, side2;
+
+        public int Side1 { get; set; }
+
+        public int Side2 { get; set; }
+
+        public int AreaCalculator(int side1, int side2)
```

Comment:
@okuzmina Kinda strange thing: If i will call these two methods with ANY 2 ints - i will get an answer, but it won't be the area of THIS PARTICULAR rectangle that we created
In these two methods you need to use not passed parameters, but the stored (in the properties Side1 and Side1!)